packages/editor-ui/src/components/mixins/pushConnection.ts

```diff
@@ -218,12 +218,33 @@ export const pushConnection = mixins(
 					// @ts-ignore
 					const workflow = this.getWorkflow();
 					if (runDataExecuted.waitTill !== undefined) {
+						const isNewWorkflow = this.$store.getters.isNewWorkflow;
+						const executionId = this.$store.getters.activeExecutionId;
+						const saveManualExecutions = this.$store.getters.saveManualExecutions;
```

Comment:
I would prefer destruction here, but also fine if you want to keep it as is

Reply:
will keep it as is... easier to search for when looking up store getters.

Reply:
updated